variables.tf

```diff
@@ -321,3 +321,44 @@ variable "max_instance_lifetime" {
   default     = 0
 }
 
+variable "instance_types" {
```

Comment:
As these are new variables, where are they set (or is this part WIP)?

Reply:
they are set in the maistro-infra repo.